src/sundials/sundials_domeigestimator.c

```diff
@@ -91,6 +92,9 @@ void SUNDomEigEstimator_FreeEmpty(SUNDomEigEstimator DEE)
   if (DEE->ops) { free(DEE->ops); }
   DEE->ops = NULL;
 
+  if (DEE->python) { free(DEE->python); }
+  DEE->python = NULL;
```

Comment:
All other frees of the python object look like

```suggestion
  free(DEE->python);
```